File: Sirma Academy May 2024/JavaScript/Conditional Statements/AlarmAfter15Minutes.js
function alarm(hour, minutes) {
  minutes += 15;

  if (minutes >= 60) {
    hour += 1;
    minutes -= 60;
  }
  if (hour >= 24) {
    hour -= 24;
  }

  let formatedHour = hour < 10 ? "0" + hour : hour;
  let formatedMinutes = minutes < 10 ? "0" + minutes : minutes;

  console.log(`${formatedHour} : ${formatedMinutes}`);
}

alarm(0, 2);

File: Sirma Academy May 2024/Java/Conditional Statements/src/AlarmAfter15Minutes.java
import java.util.Scanner;

public class AlarmAfter15Minutes {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int hour = Integer.parseInt(sc.nextLine());
        int minutes = Integer.parseInt(sc.nextLine()) + 15;

        if(minutes >= 60){
            hour += 1;
            minutes -=60;
        }
        if(hour >= 24){
            hour -=24;
        }

        System.out.printf("%02d:%02d", hour, minutes);
    }
}
